### ISSUE TITLE
Can't choose label if creating issue

### ISSUE THREAD

**Issue by OlgaKrebs** [seek4science/seek#608]
I don't mind to create without label, but it is better for developers if I could, isn't it? I asked Xiaoming why, but she suggested to ask here

**Comment by OlgaKrebs** [comment]
any answers/comments on this?

**Comment by abecam** [comment]
If you click on the gear icon you should be able to add any label you want. Is it not working?

**Comment by OlgaKrebs** [comment]
> If you click on the gear icon at the right of "Labels" you should be able to add any label you want. Is it not working?

yes, label button inactive to me. I asked Xiaoming, she told github admin can enable it ... somehow
